app/styles/base/forms.scss

```diff
@@ -296,3 +296,7 @@ input.search-input {
     border: none;
     outline: none;
 }
+
+bl-submit-btn {
```

Comment:
it is just here it will be applied to every bl-submit-btn even if it is not in a form